packages/psss-server/src/routes/reports/SubmitCountryWeeklyReportRoute.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { RespondingError } from '@tupaia/utils';
+import { Route } from '../Route';
+
+const SURVEY_CODE = 'PSSS_WNR';
+
+type WeeklyReportAnswers = {
+  PSSS_Sites: number;
+  PSSS_Sites_Reported: number;
+  PSSS_AFR_Cases: number;
+  PSSS_DIA_Cases: number;
+  PSSS_ILI_Cases: number;
+  PSSS_PF_Cases: number;
+  PSSS_DLI_Cases: number;
+};
```

Comment:
And here's the argument against defining them inline. If this was originally in the `types.ts` file, you wouldn't have recreated it here (and I think that would be a good thing)

Reply:
> Hey @rohan-bes, looking good. I'm happy with the format that you've set up here. Looks very convenient from a front end point of view. Just to check my understanding, it is as follows, right?
> 
> ### Update or Create Weekly Report Data
> METHOD: `PUT`
> ENDPOINT: `weeklyReport/TO?week=2021W07`
> JSON BODY:
> 
> ```
> {
>     "AFR": 7,
>     "DIA": 3,
>     "ILI": 9,
>     "PF": 11,
>     "DLI": 2,
>     "Sites": 13,
>     "Sites Reported": 16
> }
> ```
> 
> And it will update a record if it exists and create a new one if if it does not.

@tcaiger Yup, that's the format! However, it's simplest I think if we change the keys to match whatever works best for you on the front-end, eg.
```
{
  "afr": 7,
  "sitesReported": 12,
  ...
}
```
So if you have a preference for that just lemme know 😄

Reply:
So it's not quite the same as the other one, all the keys are different, which means I think it'd need to be defined twice anyway?
```
type ConfirmedWeeklyReportAnswers = {
  PSSS_Confirmed_Sites: number;
  PSSS_Confirmed_Sites_Reported: number;
  PSSS_Confirmed_AFR_Cases: number;
  PSSS_Confirmed_DIA_Cases: number;
  PSSS_Confirmed_ILI_Cases: number;
  PSSS_Confirmed_PF_Cases: number;
  PSSS_Confirmed_DLI_Cases: number;
};
```

I'll just confirm things with @tcaiger about the params we are passing in, and then I'll get back to you on the DRY-ness

Reply:
Oh fair! My mistake.